server/index.js

```diff
@@ -92,6 +101,28 @@ app.get('/api/v1/search', (req, res) => {
   })
 })
 
+app.put('/api/v1/me/player/play', jsonParser,(req, res) => {
+  const access_token = req.cookies.SPOTIFY_ACCESS_TOKEN
+  
+  // const search_query_parameters = new URLSearchParams({
```

Comment:
Let's remove the commented out code where we create a URLSearchParams object since we don't intend to use it for this route

Reply:
Changed this edit, thanks.